dask/dataframe/tests/test_dataframe.py

```diff
@@ -2434,6 +2434,27 @@ def test_corr():
     assert res3._name != res4._name
     assert res._name != res3._name
 
+    # Series with same names (see https://github.com/dask/dask/issues/4906)
+    a = df.A
+    b = df.B.rename('A')
+    da = dd.from_pandas(a, npartitions=6)
+    db = dd.from_pandas(b, npartitions=7)
+
+    res = da.corr(db)
+    res2 = da.corr(db, split_every=2)
+    res3 = da.corr(db, min_periods=10)
+    res4 = da.corr(db, min_periods=10, split_every=2)
+    sol = da.corr(db)
+    sol2 = da.corr(db, min_periods=10)
+    assert_eq(res, sol)
+    assert_eq(res2, sol)
```

Comment:
Seems this is failing CI.

I think a simpler test will suffice here. Something basic like

```
result = ddf.A.corr(ddf.A)
expected = df.A.corr(df.A)
assert_eq(result, expected)
```

Reply:
True. But nevertheless: any idea why it fails? I tried it locally and the values disagree by something around 0.4!